### PR TITLE
Add `access-control-allow-private-network` to the preflight

### PR DESCRIPTION
**Pull request by directionless**
See https://developer.chrome.com/blog/private-network-access-preflight/

### CODE REVIEW

File: ee/localserver/server.go [hunk 1]
```diff
@@ -308,6 +308,10 @@ func (ls *localServer) preflightCorsHandler(next http.Handler) http.Handler {
 		w.Header().Set("Access-Control-Allow-Headers",
 			"Accept, Content-Type, Content-Length, Accept-Encoding, X-CSRF-Token, Authorization")
 
+		// Some modern chrome and derivatives use Access-Control-Allow-Private-Network
+		// https://developer.chrome.com/blog/private-network-access-preflight/
```
Comment (RebeccaMahany) [inline ee/localserver/server.go:312]: https://developer.chrome.com/blog/private-network-access-update/ newer docs say this is going to be deprecated -- do we want to add a note about that?

Comment (directionless) [reply ee/localserver/server.go:312]:
```suggestion
		// https://developer.chrome.com/blog/private-network-access-preflight/
		// Though it's unclear if this is still needed, see https://developer.chrome.com/blog/private-network-access-update/
```

Comment (directionless) [reply ee/localserver/server.go:312]: Maybe like that?

Comment (RebeccaMahany) [reply ee/localserver/server.go:312]: Works for me!

**Review by RebeccaMahany — APPROVED** [review]

**Pull request merged**